app/views/merchants/show.html.erb

```diff
@@ -1,5 +1,10 @@
 <h1>Little Esty Shop</h1>
 
 <div id = "merchant">
-<%=@merchant.name %>
+ <br><h3><%=@merchant.name %></h3><br>
+</div>
+
+<div id = "merchants">
+  <h4><%=link_to "#{@merchant.name}'s Item Index", "/merchants/#{@merchant.id}/items" %></h4>
```

Comment:
I dont know if this needs to live inside of a div tag or could just go on lines 2-3, I am curious to mess around with it